xml/hwbp_registers.xml

```diff
@@ -145,6 +145,34 @@
         <field name="0" bits="2:0" access="R" reset="0" />
     </register>
 
+    <register name="Machine Context" short="mcontext" address="0x7a8">
+        This register is only writable in M mode and Debug Mode.
+
+        <field name="mcontext" bits="XLEN-1:0" access="R/W" reset="0">
+            Machine mode software can write a context number to this register,
+            which can be used to set triggers that only fire in that specific
+            context.
+
+            An implementation may tie any number of high bits in this field to
+            0. It's recommended to implement no more than 6 bits on RV32, and
+            13 on RV64.
+        </field>
+    </register>
+
+    <register name="Supervisor Context" short="scontext" address="0x7aa">
+        This register is only writable in S mode, M mode and Debug Mode.
+
+        <field name="data" bits="XLEN-1:0" access="R/W" reset="0">
+            Supervisor mode software can write a context number to this
+            register, which can be used to set triggers that only fire in that
```

Comment:
maybe "match" instead of "set"?

Reply:
match is not what I mean at all. Configure is. Is that better?